packages/extension/src/utils/format.ts

```diff
@@ -34,3 +36,13 @@ export const formatToken = (value: number, currency: string = 'XRP', isDrops = f
 
   return `${formatValue(value)} ${currency.toUpperCase()}`;
 };
+
+export const formatFlags = (flags: PaymentFlags | TrustSetFlags) => {
```

Comment:
Can you please add the missing unit test?

Reply:
Done